src/libpgmoneta/prometheus.c

```diff
@@ -2195,6 +2195,89 @@ size_information(int client_fd)
       data = NULL;
    }
 
```

Comment:
You need to define your metrics. See [here](https://github.com/pgmoneta/pgmoneta/blob/6a1ea10706c33bfdf34ed6c2b814f920100c34b4/src/libpgmoneta/prometheus.c#L570).